styles.css

```diff
@@ -42,7 +43,8 @@ body{
   margin-right: 24px;
 }
 
-.myinfo p,h1 {
+.myinfo p,
+h1 {
   flex: 0 1 100%;
```

Comment:
- [ ] You did great work, and you have added all the required changes, just one thing in your last change, your design has crashed, Kindly fix it and re-submit.

![image](https://user-images.githubusercontent.com/35535130/158722567-3f002443-dc5f-4e9b-bb63-3c14865bdca9.png)

Reply:
Hi @keroloussamy, Could you explain to me a little bit more about the problem, please?.

Reply:
@Santiago220991 The actual problem is with this style 
```
.social-media ul li img {
  width: 20px;
  height: 20px;
}
```
when you make it 20px everything is good, when you change it to 50px your design be like this 
![image](https://user-images.githubusercontent.com/35535130/158734379-eac24a96-5258-4462-b542-eb099115a6d2.png)